configman/def_sources/for_mappings.py

```diff
@@ -38,9 +38,12 @@
 
 import collections
 
-from .. import converters
-from .. import namespace
-from .. import option
+from configman.converters import str_dict_keys
+from configman.namespace import Namespace
+from configman.option import (
+    Option,
+    Aggregation,
```

Comment:
I agree with Erik Rose on this. If what you're importing and using makes sense on its own, it's fine. Like `from StringIO import StringIO`.
Unlike cases where it doesn't make sense like `from someimagelib import update`